src/assets/css/main.css

```diff
@@ -198,8 +198,7 @@ blockquote {
 
 .thumbnail {
   object-fit: cover;
-  min-height: 60px;
-  max-height: 80px;
+  aspect-ratio: 4/3;
```

Comment:
I tried this styling and the thumbnails become tiny on dektop. Is this intentional?
![image](https://github.com/LemmyNet/lemmy-ui/assets/28871516/c524c9ce-ecc0-4d6f-b986-9307055aeb72)

Reply:
Is this still happening? I can't seem to upload images to my local instance, so I can't see this, but this seems surprising to me.

Reply:
Side note – We really got to get image uploading on local instances working.

Reply:
@jsit  This was when I tried to apply your changes to my other PR. I'll have to test yours out locally.

> We really got to get image uploading on local instances working.

I can upload images locally when I use the dev docker container.

Reply:
@SleeplessOne1917 I can't reproduce the tiny image issue. Can you try again please?
<img width="860" alt="Screenshot 2023-06-26 at 6 15 26 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/a784c25e-9747-4766-aa27-779163a2072d">

Reply:
The tiny image was when I tried making some (but not the exact) changes you made in this PR to the PR I had opened earlier. This PR should be good.